tslint.json

```diff
@@ -2,9 +2,13 @@
   "rules": {
     "no-duplicate-variable": true,
     "no-unused-variable": false,
+    "no-console": [true],
     "quotemark": [true, "single"],
     "indent": [true, "spaces", 2],
     "max-line-length": [true, 120]
   },
-  "extends": "tslint-config-airbnb"
+  "extends": "tslint-config-airbnb",
+  "linterOptions": {
```

Comment:
Is disabling the linter intentional?

Reply:
Disabling the console.log check for the non-prod code

Reply:
But `*.ts` is the prod code? Or am I misunderstanding?